tardis/tardis_portal/models/storage.py

```diff
@@ -24,11 +22,19 @@ class StorageBox(models.Model):
 
     django_storage_class = models.TextField(
         default=getattr(settings, 'DEFAULT_FILE_STORAGE',
-                        'django.core.files.storage.FileSystemStorage'))
-    max_size = models.BigIntegerField()  # Bytes
-    status = models.CharField(max_length=100)
-    name = models.TextField(default='default', unique=True)
-    description = models.TextField(default='Default Storage')
+                        'django.core.files.storage.FileSystemStorage'),
+        help_text='Django storage backend. See https://docs.djangoproject.com/en/dev/ref/files/storage/')
```

Comment:
Ah OK Thanks. I just ran pep8 which must default to <80 so there was a bunch of long lines. Fixed.